config/locales/en.yml

```diff
@@ -145,6 +145,8 @@ en:
     ErrorLoggedInAsStudent: |
       You are logged in as a student in another window.
       Please log out, and register again.
+    ErrorNotAllowed: |
+      You are not allowed to perform this function.
```

Comment:
Just wondering if this could be a slightly more informative string. Maybe provide a reason why the action isn't allowed?